v19.2/use-the-built-in-sql-client.md

```diff
@@ -740,6 +741,61 @@ Time: 2.426534ms
 > SHOW DATABASE
 ~~~
 
+### Repeat a SQL statement
+
+Use the `--execute` flag to create a table from the command line:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach sql --insecure \
+--execute="CREATE TABLE t1 (id UUID DEFAULT gen_random_uuid(), name STRING)"
+~~~
+
+With the `--watch` flag, you can repeat the statements specified with the `--execute` flag periodically until an error occurs:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach sql --insecure \
+--execute="INSERT INTO t1 (name) VALUES ('a'), ('b'), ('c')" \
+--watch 3s
+~~~
+
+~~~
+INSERT 3
+INSERT 3
+INSERT 3
+INSERT 3
+INSERT 3
+^C
```

Comment:
This box is typically used to show output in our docs, but in this case the ctrl-C is shell input.  As mentioned above, I would explicitly describe the ctrl-c scenario in prose.